docs/changelog/123085.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 123085
+summary: Remove duplicated nested commands
+area: ES|QL
+type: bug
```

Comment:
definitely a bug, but it should not escape into a published artifact or release.